front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -50,6 +52,8 @@ const ManageTrainSchedule = ({ trainIdToEdit }: ManageTrainScheduleProps) => {
   const { rollingStockId, rollingStockComfort, rollingStock } =
     useStoreDataForRollingStockSelector();
 
+  const { trackSectionsLoading } = useScenarioContext();
```

Comment:
Since `trackSectionsLoading` is only used by `TimeStopInput`, and it already accesses the context for `getTrackSectionsByIds`, it would be cleaner and more maintainable to fetch it directly from the context, avoiding unnecessary prop drilling. What do you think"

Reply:
Sure! I wanted to have timesstopsinput mirror timesstopoutput, but yes it probably makes sense to put it directly in timestopsinput here